jack-sys/src/lib.rs

```diff
@@ -1084,13 +1085,23 @@ external_library!(
     fn jack_uuid_empty(jack_uuid_t) -> ::std::os::raw::c_int,
 );
 
-// Load optional functions:
-
-#[cfg(windows)]
-const jack_lib: &'static str = "libjack.dll";
-
-#[cfg(unix)]
-const jack_lib: &'static str = "libjack.so\0";
+// The following functions are not available in some JACK versions. Use with caution.
+external_library!(
```

Comment:
Does using the `external_library!` macro for the same library twice work?

Reply:
Yes, the reason its separated is because the compiled library may not include some functions. I specifically found that these are not included in the CI.

I believe the CI uses jack1 while my system uses jack2 so that might be the difference. TODO: Update the CI to test against jack1 and jack2.

Reply:
JACK1 is [deprecated](https://github.com/jackaudio/jackaudio.github.com/wiki/Differences-between-jack1-and-jack2#faq-what-are-the-differences-between-jack1-and-jack2), though if it's feasible to maintain compatibility, I don't see why not.